src/style.scss

```diff
@@ -73,12 +73,23 @@
   justify-content: space-between;
 }
 
+:host([justifyLeft]) {
+  .triggerContentWrapper {
+    justify-content: flex-start;
+  }
+
+  .triggerContent {
+    width: auto;
+  }
+}
+
 .triggerContent {
   width: 100%;
   padding-right: var(--auro-size-md);
 }
 
-:host([justifyRight]) {
+:host([justifyRight]),
+:host([justifyLeft])  {
   .triggerContent {
     padding-right: 0;
   }
```

Comment:
I find this code confusing. Looking at the layout rendered it seems this padding is to keep the text label from getting too close to the carrot/chevron.

I am not sure I understand why we want to keep them from being too close when we don't justify left/right but collapse that space when we do?

@leeejune can you comment here? Is this intended?

Reply:
The justify version is meant to have different spacing. This is outlined in the blueprint.

Reply:
There is no legacy design spec that defines this. It was just made up as we created the new versions. 

The scope of this issue is to fix the accidentally deleted references to `justifyLeft`. 

The work @leeejune is doing to redefine the blueprint spec would be applied to a new issue. The plan is to revamp the accordion element entirely.